roles/lnls-ans-role-epics/templates/epics.sh.j2

```diff
@@ -13,4 +13,4 @@ export EPICS_PVA_AUTO_ADDR_LIST="YES"
 export EPICS_EXTENSIONS=${EPICS_FOLDER}/extensions
 export EPICS_CA_MAX_ARRAY_BYTES=10000000
 
-export PATH=${PATH}:${EPICS_EXTENSIONS}
+export PATH=${EPICS_BASE}/bin/${EPICS_HOST_ARCH}:${PATH}:${EPICS_EXTENSIONS}
```

Comment:
@anacso17 I've included the epics binary directory into the path. Is it ok? I assume that we use this version as default.